mods/d2k/chrome/ingame-observer.yaml

```
@@ -708,7 +716,7 @@ Container@OBSERVER_WIDGETS:
 						ScrollItem@ECONOMY_PLAYER_TEMPLATE:
 							X: 0
 							Y: 0
-							Width: 640
+							Width: 740
```

```suggestion
							Width: 720
```

you are expanding UI by 80, not 100